src/persist/benches/benches.rs

```diff
@@ -37,6 +38,11 @@ pub fn bench_persist(c: &mut Criterion) {
     );
     writer::bench_indexed_drain(&data, &mut c.benchmark_group("writer/indexed_drain"));
     writer::bench_compact_trace(&data, &mut c.benchmark_group("writer/compact_trace"));
+
+    consensus::bench_consensus_compare_and_set(
+        &data,
```

Comment:
the defaults for this aren't really appropriate for how we expect Consensus to be used. I think it's important for `cargo bench -p mz-persist` to be meaningful without e.g. overriding `MZ_PERSIST_RECORD_SIZE_BYTES`

concretely, I propose we add a `small` method to DataGenerator like the following that has different defaults but respects the same overrides

```
impl Default for DataGenerator {
    fn small() -> Self {
        let record_count = read_env_usize(RECORD_COUNT_KEY, DEFAULT_RECORD_COUNT_SMALL);
        let record_size_bytes = read_env_usize(RECORD_SIZE_BYTES_KEY, DEFAULT_RECORD_SIZE_BYTES);
        let batch_max_count = read_env_usize(BATCH_MAX_COUNT_KEY, DEFAULT_BATCH_MAX_COUNT);

        eprintln!(
            "{}={} {}={} {}={}",
            RECORD_COUNT_KEY,
            record_count,
            RECORD_SIZE_BYTES_KEY,
            record_size_bytes,
            BATCH_MAX_COUNT_KEY,
            batch_max_count,
        );
        DataGenerator::new(record_count, record_size_bytes, batch_max_count)
    }
}
```

then construct it once here and pass it to the consensus benchmarks

this would make e.g. `MZ_PERSIST_RECORD_SIZE_BYTES=foo cargo bench -p mz-persist` not being perfectly meaningful, but i'm okay with that. i'm equally fine with introducing a `MZ_PERSIST_RECORD_SIZE_BYTES_SMALL` env if you prefer that

Reply:
done. added new environment variables to separately control the small data case.